src/test/scala/reactive5/ProductCatalogRemoteTest.scala

```diff
@@ -0,0 +1,35 @@
+package reactive5
+import akka.actor.ActorSystem
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.ConfigFactory
+import org.scalatest.{AsyncFlatSpec, Matchers}
+import reactive5.ProductCatalog.GetItems
+
+import scala.concurrent.duration._
+
+class ProductCatalogRemoteTest extends AsyncFlatSpec with Matchers {
+
+  implicit val timeout: Timeout = 3.second
+
+  "A remote Product Catalog" should "return search results" in {
+    val config = ConfigFactory.load()
+    val query  = GetItems("gerber", List("cream"))
+
+    val actorSystem = ActorSystem("ProductCatalog", config.getConfig("productcatalog").withFallback(config))
+    actorSystem.actorOf(ProductCatalog.props(new SearchService()), "productcatalog")
+
+    val anotherActorSystem = ActorSystem("another")
```

Comment:
Actor systems should be closed in `afterAll` or otherwise it is impossible to run this test 2 times in a row due to address already in use exception